Framework/API/inc/MantidAPI/IPreview.h

```diff
@@ -23,12 +23,13 @@ namespace API {
  */
 class MANTID_API_DLL IPreview {
 public:
-  enum class PreviewType { IVIEW = 0, PLOT1D = 1, PLOT2D = 2, SVIEW = 3 };
+  enum class PreviewType { IVIEW = 0, PLOT1D = 1, PLOT2D = 2, SVIEW = 3, PLOTSPECTRUM = 4 };
   virtual ~IPreview() = default;
   virtual PreviewType type() const = 0;
   virtual std::string technique() const = 0;
   virtual std::string facility() const = 0;
   virtual std::string name() const = 0;
+  virtual std::string acquisition() const = 0;
```

Comment:
I believe the acquisition mode is not used in the current version of the rawdata explorer. It is not defined in the IDF nor IPFs, and thus remains dead code. This line can be therefore removed.